src/components/filters/DateFilter/DateFilterBody/DateFilterBody.tsx

```diff
@@ -21,6 +21,11 @@ import { DateFilterHeader } from "./DateFilterHeader";
 import { DateFilterBodyButton } from "./DateFilterBodyButton";
 import { AbsolutePresetFilterItems } from "./AbsolutePresetFilterItems";
 
+const ACTIONS_BUTTONS_HEIGHT = 53;
+const EXCLUDE_OPEN_PERIOD_HEIGHT = 30;
```

Comment:
it should be `const EXCLUDE_OPEN_PERIOD_HEIGHT = 30; // height of 'Exclude open period' checkbox component`

Reply:
updated. thanks